src/views/new-evaluation/newEvaluation.js

```diff
@@ -0,0 +1,302 @@
+const NAV = function (option = {}){
+    return `
+    <ul class="header headerT">
+        ${logoButton(option.Logo)}
+        ${constructNav(option)}
+    </ul>`
+}
+
+const logoButton = function(option = {}){
+    return `
+    <li class="header-elements">
+        <img class= "main-logo" src="${option}">
+    </li>`
+}
+
+const buttonNav = function(option={}, active="" , specialPlacement = ""){
+    return `
+    <li class="header-elements ${specialPlacement}">
+        <a class = "header-elements-link header-elements-linkT ${active} " href="${option.Link}"> ${option.Text}</a>
+    </li>
+    `
+}
+
+
+const constructNav = function(option = {}){
+    const currentPage = "NewEvaluationv2.html", result =[], specialPlacement = "Loginv2.html";
+    for(var i=0;i<option.Link.length;i++){
+        if(option.Link[i] === currentPage){
+            result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "active-page"))
+        }else{ 
+            if(option.Link[i] === specialPlacement){
+                result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "", "right-button"))
+            }else{
+                result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}))
+            }
+        }
+        
+    }
+    return result.join('')
+}
+
+const candidateLine = function(option = {}){
+    return `
+    <input class="input-candidate" type="${option.DataType}" name="${option.InputName}" autocomplete="on" placeholder="${option.PlaceHolder}" required />
+    `
+}
+const constructDetailsForm = function(option = {}){
+    const result = [];
+    for(var i=0;i<option.DataType.length;i++){
+        result.push(candidateLine({DataType : option.DataType[i],
+                                InputName: option.InputName[i],
+                                PlaceHolder: option.PlaceHolder[i]}))
+    }
+    return result.join('')
+}
+
+
+const candidateDetailsForm = function (option = {}){
+    return `<form class = "candidate-input-box">
+    ${constructDetailsForm(option)}
+    </form>`
+}
+
+const technicalLevelPickerHeader = function(option = {}){
+    return ` <h3>
+        ${option}
+    </h3>`
+}
+
+
+const technicalLevelPickerTop = function(option = {}){
+    const result = []
+    for(var i=0;i<option.length;i++){
+        result.push(`<li class="R2" >${option[i]}</li>`)
+    }    
+    const Nresult = result.join('')
+    return `<ul class="top-list-radio" >
+        <li id="placeHolder"></li>
+        <li id="R1" >Trainee</li>
+        ${Nresult}
+        <li id="extra"></li>
+        <div class="clearfix"></div>
+    </ul>`
+}
+
+const technicalLevelPickerBottom = function (option = {}){
+    return ` 
+    <ul class="down-list-radio">
+        <li id= "RR1">
```

Comment:
I have to remake this part of the code. can't make it properly dynamical unless I change the whole structure :/ . Will change asap